### ISSUE TITLE
Rank bugs

### ISSUE THREAD

**Issue by IIF4D3II** [CodeCrafter47/BungeeTabListPlus#542]
Hello, I set this wonderful plugin up on my bungee network en noticed that it worked pefectly how I wanted except for 1 small thing. The rank displayname won't update on my Survival server but on the Lobby server and the Creative Server it works fine. I have only installed Essentialsx, Vault, luckperms, bungeetablistplus bukkitbridge and multichatspigot intalled on my survival server (on the lobby and creative server where it works fine are a lot more plugins). Any idea what could be the problem?
![Survival](https://user-images.githubusercontent.com/53655749/77237480-d4709280-6bc8-11ea-8050-704944d13207.png)
![Lobby](https://user-images.githubusercontent.com/53655749/77237485-d9cddd00-6bc8-11ea-8024-6fb989dbe747.png)
(Last picture is how it should be, first 1 is with the bug)

**Comment by CodeCrafter47** [comment]
I think this might be a problem with your permission setup. Have you compared the rank displayed in the tab list with the output of the `/lp` command on the different servers?

**Comment by IIF4D3II** [comment]
Yes I have done that, Luckperms is connected to my database so everything should apply the same on every server

**Comment by CodeCrafter47** [comment]
BungeeTabListPlus displays whatever information it gets from Vault (assuming you are using the Vault placeholders). If what it displays is not what you expect that points to an issue in your setup. Which you should investigate. If it is just about the tab list you can also try using a different placeholder to display the rank. E.g. there is `${viewer luckpermsbungee_primary_group}` which queries the information from LuckPerms running on BungeeCord.

**Comment by IIF4D3II** [comment]
Alright i'll look a bit deeper into my setup and give it a try. Thanks for your time!
